src/config.rs

```diff
@@ -82,6 +82,9 @@ pub struct Config<'a> {
 
     /// Ranges of lines which should be highlighted with a special background color
     pub highlighted_lines: HighlightedLineRanges,
+
+    /// The maximum number of consecutive empty lines to display
```

Comment:
Can we expand this a bit to explain what happens if it is set to `None`? Also, maybe an example wouldn't hurt? My understanding would be that `Some(3)` would mean that
```
something
like



this
```
would be printed unchanged, while
```
something
like



this
```
would be squeezed (to one empty line or to three empty lines?).

Reply:
To three empty lines.

I had a hard time trying to describe how it worked in only a few comments. The example would definitely be a good idea, thanks!